.pipelines/e2e-tests-template.yaml

```diff
@@ -48,11 +48,33 @@ jobs:
           parameters:
             identitySubscriptionID: $(SUBSCRIPTION_ID)
 
+        - script: |
+            export REGISTRY="${REGISTRY:-$(REGISTRY_NAME).azurecr.io/k8s/csi/secrets-store}"
```

Comment:
Let's remove registry here as we're running the test against the released registry which is set in the helm values.

Reply:
Done